application/listeners/posthelpers.go

```diff
@@ -260,7 +260,7 @@ func sendNotificationRequestFromCompletedToAcceptedOrDelivered(params senderPara
 }
 
 func sendNotificationEmpty(params senderParams) {
-	domain.ErrLogger.Print("Notification not implemented yet for " + params.template)
+	domain.ErrLogger.Printf("Notification not implemented yet for %s", params.template)
```

Comment:
Minor note - you could have left this the same other than adding the `f` to `Print`.  (Same for some of the others.)

Reply:
Could they have been left alone if a `Print` is defined in `ErrLogProxy`?